frontend/javascripts/admin/time/time_line_chart_view.js

```diff
@@ -0,0 +1,97 @@
+import { Chart } from "react-google-charts";
+import * as React from "react";
+
+export type ColumnDefinition = {
+  id?: string,
+  type: string,
+  role?: string,
+  p?: Object,
+};
+
+export type RowContent = [string, string, string, Date, Date];
+
+export type DateRange = [moment$Moment, moment$Moment];
+
+type Props = {
+  columns: Array<ColumnDefinition>,
+  rows: Array<RowContent>,
+  timeAxisFormat: string,
+  dateRange: DateRange,
+};
+
+export default class TimeTrackingChart extends React.PureComponent<Props> {
+  additionalCSS: ?HTMLStyleElement = null;
+  chartScrollElement: ?HTMLElement = null;
+
+  componentWillUnmount() {
+    if (this.chartScrollElement) {
+      this.chartScrollElement.removeEventListener("mousemove", this.adjustTooltipPosition);
+    }
+  }
+
+  /* We need to adjust the tooltips position manually because it is not positioned correctly when scrolling down.
+   * This fix was suggested by
+   * https://stackoverflow.com/questions/52755733/google-charts-tooltips-have-wrong-position-when-inside-a-scrolling-container.
+   * The fix is modified so that it sets the tooltip directly next to the mouse. This is done by using the total
+   * coordinates of the mouse of the whole window (clientX/Y) and manipulating the style of the tooltip directly. */
+  applyTooltipPositioningFix = () => {
+    // TimeLineGraph is the name of the chart given by the library.
+    this.chartScrollElement = document.querySelector(
+      "#TimeLineGraph > div > div:first-child > div",
+    );
+    if (this.chartScrollElement) {
+      this.chartScrollElement.addEventListener("mousemove", this.adjustTooltipPosition);
+    }
+  };
+
+  adjustTooltipPosition = (event: MouseEvent) => {
+    const tooltip = document.getElementsByClassName("google-visualization-tooltip")[0];
+    if (tooltip != null) {
+      tooltip.style.top = `${event.clientY}px`;
+      tooltip.style.left = `${event.clientX + 15}px`;
+      // We need to make the tooltip visible again after adjusting the position.
+      // It is initially invisible as it is mispositioned by the library and would then "beam" to the corrected
+      // position. We want to avoid that "beaming" behaviour.
+      tooltip.style.visibility = "visible";
+    }
+  };
+
+  render() {
+    const { columns, rows, timeAxisFormat, dateRange } = this.props;
+
+    const { applyTooltipPositioningFix } = this;
+
+    return (
+      <Chart
+        chartType="Timeline"
+        columns={columns}
+        rows={rows}
+        options={{
+          timeline: { singleColor: "#108ee9" },
+          // Workaround for google-charts bug, see https://github.com/scalableminds/webknossos/pull/3772
+          hAxis: {
+            format: timeAxisFormat,
+            minValue: dateRange[0].toDate(),
+            maxValue: dateRange[1].toDate(),
+          },
+          allowHtml: true,
+        }}
+        graph_id="TimeLineGraph"
+        chartPackages={["timeline"]}
+        width="100%"
+        height="600px"
```

Comment:
@philippotto: We need a fixed height as the workaround we found, that displays the chart at full height, leaves out the whole y-axis with the actual dates. Having a fixed height solves this. 
I also shortly tried to force the y-axis to be rendered below the chart but failed :/